.studiorc

```diff
@@ -273,6 +273,35 @@ origin() {
   fi
 }
 
+test-builder() {
+    hab sup status
+    if [ $? -eq 0 ]; then
+        echo ""
+        echo "*****************"
+        echo "Running supervisor detected"
+        echo "*****************"
+        echo ""
+        echo "*****************"
+        echo "Terminating running supervisor"
+        echo "*****************"
+        echo ""
+        hab sup term
+        sleep 8
+    fi
+
+    echo ""
+    echo "*****************"
+    echo "Restarting supervisor in test mode"
+    echo "*****************"
+    echo ""
+    HAB_FUNC_TEST=1 sup-run
+    sleep 10
+    echo "*****************"
+    echo "Executing functional tests"
+    echo "*****************"
+    /src/test/builder-api/test.sh
+}
```

Comment:
Should we stop test mode after the tests have run?

Reply:
Good call. Added.